provider-service/kfp/cmd/main.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"context"
+	"github.com/go-logr/logr"
+	"github.com/go-resty/resty/v2"
+	"github.com/sky-uk/kfp-operator/argo/common"
+	. "github.com/sky-uk/kfp-operator/provider-service/base/pkg"
+	configLoader "github.com/sky-uk/kfp-operator/provider-service/base/pkg/config"
+	"github.com/sky-uk/kfp-operator/provider-service/base/pkg/streams/sinks"
+	"github.com/sky-uk/kfp-operator/provider-service/base/pkg/streams/sources"
+	kfp "github.com/sky-uk/kfp-operator/provider-service/kfp/internal"
+	"go.uber.org/zap/zapcore"
+	"os"
+)
+
+func main() {
+	logger, err := common.NewLogger(zapcore.InfoLevel)
+	if err != nil {
+		panic(err)
+	}
+	ctx := logr.NewContext(context.Background(), logger)
+
+	config, err := configLoader.LoadConfig(ctx)
+	if err != nil {
+		logger.Error(err, "Failed loading configuration")
+		panic(err)
```

Comment:
`panic` or `os.Exit(1)` should probably decide and be consistent throughout this file and the other `main.go` for `vai`

Reply:
changed in 847762891ac9e4c53e697be2fbd16d03bfb70260